layout/header/component.tsx

```diff
@@ -3,13 +3,14 @@ import React, {
 } from 'react';
 
 // components
+import LogIn from 'components/log-in';
```

Comment:
I would suggest renaming the component to `userDropdown` or similar. `LogIn` can be confusing given we already have a `SignIn` around.